apiary.apib

```diff
@@ -4,7 +4,107 @@ FORMAT: 1A
 
 The API for interacting with IPFS nodes.
 
-# Group add
+# Group add [POST /add{?arg}{&recursive,quiet,silent,progress,trickle,only%2Dhash,wrap%2Dwith%2Ddirectory,hidden,chunker,pin}]
```

Comment:
I would be much in favor of dropping support for the shorthands like `r` and `q` as they are just very cryptic and `q` has a different meaning in most http based APIs as `query`.